src/index.html

```diff
@@ -4,9 +4,443 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo logo">
+        <a class ="logo__image" href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__link link">
+            <a
+              class="link__adress is-active"
```

Comment:
class="nav__adress"
and in Sass it will be really easy to write styles for this block.
check all li > a

Reply:
i have this problem
![image](https://user-images.githubusercontent.com/21061927/186171080-957e57d6-64f9-49a0-bc80-ae15398d752e.png)